blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -92,6 +92,9 @@ Changes
 Fixes
 =====
 
+- Fixed an issue with the ``collect_set`` function. It could compute an
```

Comment:
No `:ref:` because it is not documented, at the same time we already backported other `collect_set` related fixes.